cbrf/const.py

```diff
@@ -1,14 +1,20 @@
 # -*- coding: utf-8 -*-
+import os
 
 CURRENCY_CODES = {
     # http://www.cbr.ru/scripts/XML_val.asp?d=0
     'USD': 'R01235',
     'EUR': 'R01239',
 }
 
+CBRF_URL_SCHEME = os.getenv('CBRF_URL_SCHEME', 'http')
```

Comment:
SCHEMA should be replased by "PROTOCOl"

Reply:
Scheme seem to be correct for URI
https://tools.ietf.org/html/rfc3986#section-3.1